1_Lyalyushkin/CoreFoundation.py

```diff
@@ -0,0 +1,79 @@
+import ctypes
+from ctypes.util import find_library
+from ctypes import c_void_p, c_char_p, c_uint32, POINTER, c_long
+
+cf_path = find_library('CoreFoundation')
+CoreFoundation = ctypes.cdll.LoadLibrary(ctypes.util.find_library('CoreFoundation'))
+
+CFIndex = c_long
+CFStringEncoding = c_uint32
+CFString = c_void_p
+CFArray = c_void_p
+CFDictionary = c_void_p
+CFError = c_void_p
+CFType = c_void_p
+
+CFAllocatorRef = c_void_p
+CFStringRef = POINTER(CFString)
+CFArrayRef = POINTER(CFArray)
+CFDictionaryRef = POINTER(CFDictionary)
+CFErrorRef = POINTER(CFError)
+CFTypeRef = POINTER(CFType)
+
+CoreFoundation.CFStringGetCStringPtr.argtypes = [CFStringRef, CFStringEncoding]
+CoreFoundation.CFStringGetCStringPtr.restype = c_char_p
+
+CoreFoundation.CFStringGetCString.argtypes = [CFStringRef, c_char_p, CFIndex, CFStringEncoding]
+CoreFoundation.CFStringGetCString.restype = ctypes.c_bool
+
+CoreFoundation.CFStringCreateWithCString.argtypes = [CFAllocatorRef, c_char_p, CFStringEncoding]
+CoreFoundation.CFStringCreateWithCString.restype = CFStringRef
+
+CoreFoundation.CFArrayGetCount.argtypes = [CFArrayRef]
+CoreFoundation.CFArrayGetCount.restype = CFIndex
+
+CoreFoundation.CFArrayGetValueAtIndex.argtypes = [CFArrayRef, CFIndex]
+CoreFoundation.CFArrayGetValueAtIndex.restype = CFTypeRef
+
+CoreFoundation.CFDictionaryGetValue.restype = CFStringRef
+CoreFoundation.CFDictionaryGetValue.argtypes = [CFDictionaryRef, CFStringRef]
+
+kCFStringEncodingUTF8 = CFStringEncoding(0x08000100)
```

Comment:
What is the meaning of magic number `0x08000100`? Can it be defined as pseudo-constant with some clear name?